docs/oxidized-hook

```diff
@@ -0,0 +1,22 @@
+This assumes using oxidized with an git basted storage, it might look little different if you use flat-file storage.
```

Comment:
Can you convert this to [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) so Sphinx can deal with it? That would allow it to go into the [web documentation](https://netlint.readthedocs.io). Thank you!

Reply:
Ah, seen a markup like that before (quite common in git) 
Will do.